tests/self/DGMax/DGmax_Vacuum2D_SelfTest.cpp

```diff
@@ -147,5 +169,19 @@ int main(int argc, char** argv) {
             0.01, true);  // Small eigenvalues are to be expected from DGMax
         result2.printFrequencyTable(linear);
         result2.printErrorTable(linear);
+
+        // Some spacing between the results for the two seperate tests
+        std::cout << "\n\n";
+
+        std::cout << "Raw frequency table 3" << std::endl;
+        result3.printFrequencyTable(linear);
+
+        result3.printResultCode(10);
+
+        std::cout << "Cleanup results" << std::endl;
+        result3.filterResults(
```

Comment:
Are there? Because the lowest eigenvalue that it should find is `0.8^2 + 0.9^2`. I would expect all zero eigenvalue to be filtered out.